ts/packages/ts/src/commands/EvalCommand.ts

```diff
@@ -224,6 +227,14 @@ export class EvalCommand extends Command<EvalCommandReturn> {
     return resultWithTemplates;
   }
 
+  private removeForwardsFromPipe(result: EvalCommandReturn): EvalCommandReturn {
```

Comment:
The naming of the method is a bit misleading in my opinion because we perform the removal from the result and not the pipe itself. How about one of these?
- `removeForwardsFromPrevious` -> aligns with `evalPrevious`
- `removeForwards`

Reply:
1. if you take a look at the method's logic, `pipe[current.context.key]` validate we're only removing from the piped configs
2. removeForwardsFromPrevious -> aligns with evalPrevious i don't understand what you mean here
3. sure. `removeForwards` it is

Reply:
1. Was true until you showed me the forward chain bug. I'll revise. Thanks